spec/docx/regression/api_range_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+describe 'ApiRange regression tests' do
+  it 'ApiRange | 71120 IntersectWith method' do
+    skip('https://bugzilla.onlyoffice.com/show_bug.cgi?id=71590')
```

Comment:
Why skip it if the bug is fixed in version 8.2.1? Maybe it's worth adding a version check?

Reply:
It's another bug =)